lib/mm_network.pm

```diff
@@ -172,6 +173,10 @@ sub check_ip_in_subnet {
 
 sub setup_static_mm_network {
     my $ip = shift;
+    if (is_opensuse) {
+        assert_script_run "systemctl disable NetworkManager --now";
```

Comment:
Hmm, as we have same issue with yast cmd tests on TW and we agreed to create image with wicked enabled, I wonder if we should just use that image then? WDYT?
Obviously, we can have it like this for now and remove once we have image, I will update [poo#64731](https://progress.opensuse.org/issues/64731) with these details.

Reply:
I would just have the cmd test suite depend on create_hdd_textmode, as we have wicked by default when there's no desktop